### DISCUSSION TITLE
approve Glossary term

### DISCUSSION THREAD

**Discussion by tm-yegorova** [open-metadata/OpenMetadata#11577] [General]
Hi, I need that reviewers of glossary terms could approve it.
As I understood There isn't such function.
Please give me advice how to do it, may be using API. Really it's a proplem that we haven't opportunity to aprove term and genarate alerts about new not approved glossary terms for reviewers.
It would be better for me to do it using python or just configurate something, but I have to use any variant you  suggest.
If you know code here to help me, lease share.

**Comment by harshach** [comment]
This is a pending feature we haven't implemented yet. Its targetted for 1.2 release

**Reply by tm-yegorova** [reply]
Thankyou for your answer. When is release 1.2 expected?

**Reply by dao2000** [reply]
Hello! Please tell me the time frame when the functionality of the possibility of approving glossary terms by visors will appear. Thanks!

**Comment by harshach** [comment]
@tm-yegorova @dao2000 it will be part of 1.2 release , Aug 22nd tentatively
